test/ServerTest.cpp

```diff
@@ -12,23 +12,25 @@
 #include "util/http/HttpUtils.h"
 #include "util/http/UrlParser.h"
 
-namespace http = boost::beast::http;
-using namespace ad_utility;
+namespace m {
```

Comment:
For such functions (only local to a `cpp` file, not meant to be called from other translation units) the correct way is to either make them `static` or (we typically use the second approach, as it also supports classes/typedefs etc)
to put them into the anonymous namespace, so
```
namespace {
auto ParsedRequest = ....;
} 

// use `ParsedRequest` but it can under no circumstances be used from a different cpp file, and can never lead to ODR violations.